website/project/metadata/schemas.py

```diff
@@ -1,17 +1,11 @@
 import os
 import json
 
+
 def ensure_schema_structure(schema):
-    if 'pages' not in schema['schema']:
-        schema['schema'] = {
-            'pages': [
-                {
-                    'id': 'page1',
-                    'title': '',
-                    'questions': schema['schema']['questions'],
-                }
-            ]
-        }
+    schema['pages'] = schema.get('pages', [])
+    schema['title'] = ' '.join(schema.get('name', '').split('_'))
```

Comment:
If I'm reading this right, this seems to be equivalent to `'my_string'.replace('_', ' ')`. Try to use simpler string methods if possible, else add a comment explaining the more convoluted solution.